js/main.js

```diff
@@ -15,3 +80,105 @@ navItem.forEach((n) => {
     document.body.classList.remove('hidden-flow');
   });
 });
+
+const createProject = (project) => {
+  const projectEl = document.createElement('div');
+  projectEl.classList.add('project');
+
+  const img = document.createElement('img');
+  img.src = project.featuredImage;
+
+  const name = document.createElement('h3');
+  name.classList.add('project-heading');
+  name.innerText = project.name;
+
+  const ul = document.createElement('ul');
+  ul.classList.add('techs');
+
+  project.technologies.forEach((el) => {
+    const li = document.createElement('li');
+    li.classList.add('techs-item');
+
+    li.innerText = el;
+    ul.appendChild(li);
+  });
+
+  const btn = document.createElement('button');
+  btn.classList = 'btn see-project';
+  btn.id = project.id;
+  btn.innerText = 'See project';
+
+  projectEl.appendChild(img);
+  projectEl.appendChild(name);
+  projectEl.appendChild(ul);
+  projectEl.appendChild(btn);
+
+  return projectEl;
+};
+
+// const modalContainer = document.querySelector('.modal-container')
+const renderPopup = (project) => `
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. :slightly_smiling_face: